homeassistant/components/cast/media_player.py

```diff
@@ -616,6 +656,7 @@ def audio_content_filter(item):
             self.hass, media_content_id, content_filter=content_filter
         )
 
+    @api_error
```

Comment:
Will the decorator work for a coroutine function?

Reply:
Well spotted, thanks a lot! 👍 
No, I don't think it will work as intended. I'll extract the calls to the cast API to helpers wrapped with `@api_error`.